packages/theme/src/pageTheme.ts

```diff
@@ -63,4 +63,9 @@ export const pageTheme: Record<string, PageTheme> = {
   library: genPageTheme(colorVariants.rubyRed, shapes.wave),
   other: genPageTheme(colorVariants.darkGrey, shapes.wave),
   app: genPageTheme(colorVariants.toastyOrange, shapes.wave),
+  apis: genPageTheme(colorVariants.eveningSea, shapes.wave2),
+  grpc: genPageTheme(colorVariants.marineBlue, shapes.wave2),
```

Comment:
Let's skip the themes for the specific API types for now, sticking to just adding the `apis` theme.

Reply:
done

Reply:
Still seeing them here 😅